core/src/main/scala/kafka/raft/KafkaNetworkChannel.scala

```diff
@@ -53,6 +54,41 @@ object KafkaNetworkChannel {
 
 }
 
+private[raft] class RaftSendThread(
```

Comment:
style nit/question: I think we have a mixture of argument indentation for Scala classes/methods. Do we have an established style convention for this? Normally I follow the opening paren when breaking out arguments into their own line (though I'm not sure that's correct)

Reply:
Not really as far as I know. This is my favored style of late because it results in consistent alignment of parameters. I get really annoyed when I see stuff like this:
```
def someMethodWithAnArguablyOverVerboseName(foo: String,
                                            bar: Option[List[Int]]): (Option[String], List[String)

def concicseMethod(foo: String,
                   bar: String): Unit = {
```
No matter how you look at it, it seems hideous.